yewprint-doc/src/app.rs

```diff
@@ -322,6 +330,8 @@ pub enum DocMenu {
     Tag,
     #[to = "/#text"]
     Text,
+    #[to = "/#textarea"]
+    TextArea,
```

Comment:
```suggestion
    #[to = "/#textarea"]
    TextArea,
    #[to = "/#text"]
    Text,
```

It stops matching the URL at first match

Reply:
Addressed.